src/components/MainContent/PostList/PostItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useContext } from 'react';
+import { Post } from '../../../types/Post';
+import classNames from 'classnames';
+import {
+  ActivePostContext,
+  CommentFormContext,
+  CommentListContext,
+  ErrorsContext,
+  LoaderContext,
+} from '../../../utils/Store';
+import { getComments } from '../../../utils/fetchFunctions';
+import { ErrorTypes } from '../../../types/ErrorTypes';
+
+interface PostItemProps {
+  post: Post;
+}
+
+export const PostItem: React.FC<PostItemProps> = ({ post }) => {
+  const { activePost, setActivePost } = useContext(ActivePostContext);
+  const { setIsLoading } = useContext(LoaderContext);
```

Comment:
Consider renaming 'setIsLoading' to 'setLoading' or 'setLoaded' to follow the checklist requirement: 'don't use isLoad, it can be isLoading or isLoaded'.